package.json

```diff
@@ -2,32 +2,45 @@
   "name": "yeoman-assert",
   "version": "2.1.1",
   "description": "Assert utility from yeoman",
-  "license": "BSD",
+  "homepage": "http://yeoman.io",
+  "author": "The Yeoman Team",
+  "files": [
+    "index.js"
+  ],
+  "main": "index.js",
   "keywords": [
     "assert",
     "utility",
     "helper",
     "yeoman"
   ],
-  "homepage": "http://yeoman.io",
-  "author": "The Yeoman Team",
+  "license": "BSD",
```

Comment:
Oh, this is BSD-2-Clauses - see https://github.com/yeoman/generator/blob/master/package.json#L25